src/Resources/Products/Products.ts

```diff
@@ -0,0 +1,24 @@
+import AbstractResource from '../AbstractResource';
+
+export default class Products extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '/products';
+        this.defaultAccess = 'guest';
+    }
+
+    /**
+     * **Main endpoint for browse toys.**
```

Comment:
Comment is malformed

Reply:
It's meant to be like that, markdown to make it bold.
![Screenshot 2020-04-28 at 16 04 09](https://user-images.githubusercontent.com/15678246/80503547-eb459a00-8969-11ea-95ff-6e1169a5ae0c.jpg)

Reply:
Very interesting, thanks for showing me that